csvkit/cli.py

```diff
@@ -245,6 +248,8 @@ def _init_common_parser(self):
                 help='Insert a column of line numbers at the front of the output. Useful when piping to grep or as a '
                      'simple primary key.')
 
+        AddBOM.argument(self.argparser,self)
```

Comment:
Please conform to the existing code, instead of introducing an entirely different code organization pattern.